source/libs/page-detect.js

```diff
@@ -61,6 +61,8 @@ export const isMilestoneList = () => /^milestones\/?$/.test(getRepoPath());
 
 export const isNewIssue = () => /^issues\/new/.test(getRepoPath());
 
+export const isNewPR = () => /^compare\/.+$/.test(getRepoPath());
```

Comment:
We already have a `isCompare` test. This also includes pages like

https://github.com/parcel-bundler/parcel/compare/v1.4.0...v1.4.1

Sadly there's no way to distinguish compare-only pages to `newPR` pages, especially because the form can be opened later: https://github.com/parcel-bundler/parcel/compare/master...vue